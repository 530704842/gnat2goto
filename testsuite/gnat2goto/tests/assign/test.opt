ALL XFAIL CBMC crashes on this
