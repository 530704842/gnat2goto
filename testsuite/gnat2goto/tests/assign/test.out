[overflow.1] arithmetic overflow on signed + in assign__a + assign__b: SUCCESS
VERIFICATION SUCCESSFUL
